pyingest/parsers/jats.py

```diff
@@ -388,6 +407,16 @@ def parse(self, fp, **kwargs):
                         base_metadata['pubdate']
                     except Exception, err:
                         base_metadata['pubdate'] = pubdate
+            try:
+                if (b == 'open-access'):
+                    try:
+                        base_metadata['properties']
+                    except:
+                        pass
+                    else:
+                        base_metadata['properties']['OPEN'] = ''
```

Comment:
should be:
`base_metadata['properties']['OPEN'] = 1`

Reply:
I'm not creating the OPEN key at all if it's not open access -- please confirm this is correct.

Reply:
The code as written is not setting the properties to open if `base_metadata['properties']` does not already exist.  Also I believe we want to assign the value of '1' or True to `base_metadata['properties']['OPEN']`.  I suggest this code:
```
if b == 'open-access':
    base_metadata.setdefault('properties',{}).setdefault('OPEN',1)
```